### PR TITLE
Updated tests to pass with current MAA service

### PR DESCRIPTION
**Pull request by LarryOsterman**

### CODE REVIEW

**Comment by LarryOsterman** [comment]
Ping :).

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -23,6 +23,12 @@
 /sdk/appconfiguration/azure-spring-cloud-*/         @mrm9084 @zhenlan @saragluna @yiliuTo @chenrujun @backwind1233 @stliu
 /sdk/appconfiguration/spring-cloud-azure-*/         @mrm9084 @zhenlan @saragluna @yiliuTo @chenrujun @backwind1233 @stliu
 
+
+# PRLabel: %Attestation
+/sdk/attestation/                                   @azure-sdk-write-attestation @larryosterman
+/sdk/attestation/azure-security-attestation         @azure-sdk-write-attestation @larryosterman
+/sdk/attestation/azure-resourcemanager-attestation         @azure-sdk-write-attestation @larryosterman
```
Comment (pallavit) [inline .github/CODEOWNERS:30]: [nit[ spacing.

**Review by pallavit — APPROVED** [review]
:shipit:

**Pull request merged**